Source/WebInspectorUI/UserInterface/Controllers/ApplicationCacheManager.js

```diff
@@ -1,274 +0,0 @@
-/*
- * Copyright (C) 2013 Apple Inc. All rights reserved.
- *
- * Redistribution and use in source and binary forms, with or without
- * modification, are permitted provided that the following conditions
- * are met:
- * 1. Redistributions of source code must retain the above copyright
- *    notice, this list of conditions and the following disclaimer.
- * 2. Redistributions in binary form must reproduce the above copyright
- *    notice, this list of conditions and the following disclaimer in the
- *    documentation and/or other materials provided with the distribution.
- *
- * THIS SOFTWARE IS PROVIDED BY APPLE INC. AND ITS CONTRIBUTORS ``AS IS''
- * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO,
- * THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
- * PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL APPLE INC. OR ITS CONTRIBUTORS
- * BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
- * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
- * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
- * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
- * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
- * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF
- * THE POSSIBILITY OF SUCH DAMAGE.
- */
-
-// FIXME: ApplicationCacheManager lacks advanced multi-target support. (ApplciationCache objects per-target)
-
-WI.ApplicationCacheManager = class ApplicationCacheManager extends WI.Object
-{
-    constructor()
-    {
-        super();
-
-        this._enabled = false;
-        this._reset();
-    }
-
-    // Agent
-
-    get domains() { return ["ApplicationCache"]; }
-
-    activateExtraDomain(domain)
-    {
-        // COMPATIBILITY (iOS 14.0): Inspector.activateExtraDomains was removed in favor of a declared debuggable type
-
-        console.assert(domain === "ApplicationCache");
-
-        for (let target of WI.targets)
-            this.initializeTarget(target);
-    }
-
-    // Target
-
-    initializeTarget(target)
-    {
-        if (!this._enabled)
-            return;
-
-        if (target.hasDomain("ApplicationCache")) {
-            target.ApplicationCacheAgent.enable();
```

Comment:
please also remove this from `Source/WebInspectorUI/UserInterface/Protocol/Target.js`
```js
get ApplicationCacheAgent() { return this._agents.ApplicationCache; }
```

generally speaking, im slightly hesitant about removing this from the UI as it means we'll be dropping support when the backend technically could still work, but seeing as how this all has been disabled for so long i think it's fine

if we are gonna remove the frontend bits, however, we should manually edit all of the `Source/WebInspectorUI/UserInterface/Protocol/Legacy/*/*/InspectorBackendCommands.js` to not have the `ApplicationCache` domain as otherwise we'll still create the `WI.ApplicationCacheObserver` class, `ApplicationCacheAgent` class, etc. which is just wasteful

Reply:
I will update the legacy protocol to remove it.